commit 20e606cedfe51bf89ceade0ea3497c0099bf8240
Author: Apollon Oikonomopoulos <apoikos@gmail.com>
Date:   Fri Jul 11 12:37:12 2014 +0300

    Do not backup exported instance data on upgrade

    gnt-cluster upgrade creates a configuration backup by tar'ing
    /var/lib/ganeti. Unfortunately, this means that exported instance data
    is also backed up during upgrade, which is clearly unwanted.

    We have been using /var/lib/ganeti/export for exported instance data
    since 2.0.1-1 and changing the location now will break imports in
    existing setups, so it's best to just skip backing up DATADIR/export.

--- a/lib/client/gnt_cluster.py
+++ b/lib/client/gnt_cluster.py
@@ -1956,6 +1956,7 @@
   (_, tmp_name) = tempfile.mkstemp(prefix=backuptar, dir=pathutils.BACKUP_DIR)
   if not _RunCommandAndReport(["tar", "-cf", tmp_name,
                                "--exclude=queue/archive",
+                               "--exclude=export",
                                pathutils.DATA_DIR]):
     return (False, rollback)
 
